setup.py

```diff
@@ -47,9 +47,9 @@
     ],
     extras_require={
         "gui": [
-            "pyside6<6.3.2",
+            "pyside6==6.4.2",
             "qdarkstyle==3.1",
-            "napari-deeplabcut>=0.2.1.2",
+            "napari-deeplabcut>=0.2.1.4",
```

Comment:
Update for the [latest release](https://pypi.org/project/napari-deeplabcut/#history)

```suggestion
            "napari-deeplabcut>=0.2.1.5",
```